### DISCUSSION TITLE
Thank you for picking up z-push and zimbra backend! I know a few people in perticular showing up for this for years.

### DISCUSSION THREAD

**Discussion by tuxcrafter** [Z-Hub/Z-Push#128] [General]
It is a shame what happened with Kopano and the lack of security and features updates in there repositories.

I am willing to donate vms/hosting for new repos. There was a lot of goodness in those RPM/DEP packages that I am missing like, rolling security trough repos, php-awl, selinux configs, httpd configs, correct permissions, log settngs, etc etc.

I got zimbra 10 OSE zcs-10.0.0_FOSS_intalio.UBUNTU20_64 working with zpush 2.7.5 with zimbrabackend zimbra73 and saml-zm-sso

Spending load of resources just to stay secure and loosing features not a fun way to spend our energy.

PS not a clue what to do with  https://www.davical.org/downloads/awl_0.64.orig.tar.xz as there is no rpm version of this to be found. I was using RHEL7 repo from kopano but the signing key is SHA1 insecure and not compatible with Rocky9.

Thank you all!

**Comment by matidau** [comment]
No problems. You should probably pass on your thanks for the Zimbra Backend to @liverpoolfc-fan at https://sourceforge.net/p/zimbrabackend/discussion/998911/ this is where the backend is maintained.
